sass/_page.sass

```diff
@@ -237,6 +237,93 @@ button
         &.underline-from-right
             @include underline(right, $activeColor, 5px)
 
+
+        // Icon Transitions
+        &.icon-back
+            @include icon-back
```

Comment:
Can you use a loop to make this simpler to add others?
```sass
$icons: back, bob, bounce-out, etc
@each $icon in $icons
    &.icon-#{$icon}
        @include icon-#{$icon}
```
or maybe that won't work, but worth trying I think